Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1616,8 +1616,8 @@ void RenderGrid::layoutMasonryItems(GridLayoutState& gridLayoutState)
         gridItem.layoutIfNeeded();
 
         // We need pending layouts to be done in order to compute auto-margins properly.
-        updateAutoMarginsInColumnAxisIfNeeded(gridItem);
-        updateAutoMarginsInRowAxisIfNeeded(gridItem);
+        GridLayoutFunctions::updateAutoMarginsInColumnAxisIfNeeded(gridItem, writingMode());
+        GridLayoutFunctions::updateAutoMarginsInRowAxisIfNeeded(gridItem, writingMode());
```

Comment:
Might be good to store the writing mode in a local variable so we could avoid calling the function twice